lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/API/StakePools.hs

```diff
@@ -99,3 +107,93 @@ spec = do
             r <- request @ApiStakePool ctx (method, "v2/stake-pools") Default Empty
             expectResponseCode @IO HTTP.status405 r
             expectErrorMessage errMsg405 r
+
+    it "STAKE_POOLS_JOIN_01 - Can join a stakepool" $ \ctx -> do
+        (_, p:_) <- eventually' $ unsafeRequest @[ApiStakePool] ctx listStakePoolsEp Empty
```

Comment:
```suggestion
        (_, p:_) <- eventually' $
            unsafeRequest @[ApiStakePool] ctx listStakePoolsEp Empty
```

https://github.com/input-output-hk/cardano-wallet/wiki/Coding-Standards#limit-line-length-to-80-characters